src/components/Molecules/NavbarLeft/NavbarLeft.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { render, cleanup } from '@testing-library/react';
```

Comment:
Import `screen` from `@testing-library/react` and use it to run your RTL queries.

Reply:
Replaced all ReactDOM by "screen" from RTL